pom.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 /********************************************************************************
- * Copyright (c) 2023 T-Systems International GmbH
- * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ * Copyright (c) 2023, 2024 T-Systems International GmbH
```

Comment:
Please remove the whitespaces between the years according to [TRG7.02](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02#copyright-and-license-header).